<?xml version="1.0"?>
<launch>
    <arg name="calib_type" default="aruco_board"/> <!--Supports "aruco_board" or "chessboard"-->
    <node name="panda_autograsp_server" pkg="panda_autograsp" type="panda_autograsp_server.py" output="screen">
        <param name="calib_type" type="str" value="$(arg calib_type)"/>
        <remap from="image_color" to="kinect2/hd/image_color"/>
        <remap from="image_color_rect" to="kinect2/sd/image_color_rect"/>
        <remap from="image_depth_rect_32FC1" to="kinect2/sd/image_depth_rect_32FC1"/>
        <remap from="hd/camera_info" to="kinect2/hd/camera_info"/>
        <remap from="qhd/camera_info" to="kinect2/qhd/camera_info"/>
        <remap from="sd/camera_info" to="kinect2/sd/camera_info"/>
    </node>
</launch>